scripts/monitoring/cron-send-rpms-versions.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Check OpenShift version
+OC_VERSION=$(oc version | sed -n '1{p;q}' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
```

Comment:
We currently bind in the oc binary, but we may not always do that.

I think this should probably talk to the openshift api and try to get the version from there.

I'm not 100% sure if it's there or not, but that might be best.

Failing that, this might be the only other reasonable way to get this value.